lib/virtfs/xfs.rb

```diff
@@ -1 +1,20 @@
 require "virtfs/xfs/version"
+require_relative "xfs/allocation_group"
+require_relative "xfs/bmap_btree_block"
+require_relative "xfs/bmap_btree_record"
+require_relative "xfs/bmap_btree_root_node"
+require_relative "xfs/directory"
+require_relative "xfs/directory2_data_header"
+require_relative "xfs/directory3_data_header"
+require_relative "xfs/directory_block_tail"
+require_relative "xfs/directory_data_header"
+require_relative "xfs/directory_entry"
+require_relative "xfs/inode"
+require_relative "xfs/inode_map"
+require_relative "xfs/short_form_directory_entry"
+require_relative "xfs/short_form_header"
+require_relative "xfs/superblock"
+
+require_relative "xfs/base"
+require_relative "xfs/file_class"
+require_relative "xfs/dir_class"
```

Comment:
@Fryguy I have no problem with your suggestion other than the fact that the VirtFS written by @roliveri and three PRs for VirtFS-NTFS, VirtFS-Ext3, and VirtFS-Ext4 by @movitto do it this way - I just used the same pattern.  We should either change them all or keep them all this way.